wiki/en/en-Client-Troubleshooting.md

```diff
@@ -7,8 +7,8 @@ permalink: "/wiki/Client-Troubleshooting"
 
 # Troubleshooting
 
-### Don’t hear any sound/Others don’t hear you?
-Start with the simple stuff: make sure your instrument/microphone and headphones are connected to the correct sockets. Make sure no other applications like your browser, video conferencing tool etc. is also using your sound card. You should shut those down when using Jamulus. If that all looks OK and the problem persists, it's likely there's an issue with your sound device settings. This will depend on your particular setup (platform, hardware, software and drivers), so it's best to ask on [the forums](https://github.com/jamulussoftware/jamulus/discussions) for guidance.
+### Don’t hear any sound/Others don’t hear you or you get an error-message while connecting to a server?
+Start with the simple stuff: if you are using an USB-microphone, connect it directly to an USB-port in your computer and avoid using an USB-hub. Even newer USB-hubs have caused the error message "Your sound card is not working correctly. Please open the settings dialog and check the device selection and the driver settings.". Make sure your instrument/microphone and headphones are connected to the correct sockets. Make sure no other applications like your browser, video conferencing tool etc. is also using your sound card. You should shut those down when using Jamulus. If that all looks OK and the problem persists, it's likely there's an issue with your sound device settings. This will depend on your particular setup (platform, hardware, software and drivers), so it's best to ask on [the forums](https://github.com/jamulussoftware/jamulus/discussions) for guidance.
```

Comment:
When saying "if you are using an USB-microphone, connect it directly to an USB-port in your computer and avoid using an USB-hub." then the second sentence explaining why is unnecessary because the user can't act on that information. We don't explain why, for example, a video conferencing tool should be shut down.

Reply:
Is updated.

Reply:
It should be **a** USB, not **an**. Although it is a vowel, the 'U' is pronounced like a 'Y', which is a consonant.

Reply:
Please refere to my latest PR-update.

Reply:
I'm still not sure what the user is supposed to do with the information about the error message. Forgive me, but it just seems like noise.

Reply:
That "noice" would have saved me hours of troubleshooting with an somehow unhelpfull error message. The point is, ONLY jamulus seems to complain. Having a working setup in Cubase, Zoom, BigBlueButton and many others but getting error messages in jamulus complaining about "Soundcard not working correctly" does obviously not point to that kind of problem. Isn't a "troubleshoot page" just made to give hints what to try out to solve a problem?
What I would have done with this input on the page? I would have tried it and in seconds have the result: my USB-hub just sucks.

Reply:
Agree. That's why I also think it should be on this page ;-). Nevertheless this message is about a generic error. See my other comment

Reply:
OK but if we have to have this, it feels to me like it's a separate section in the doc. It's not _"Don’t hear any sound/Others don’t hear you?"_ it's _"Getting the message 'Your sound card is not working correctly' and don't know why?"_ isn't it?

We need to fix the bug that gives rise to this too, so having it as a separate item makes it easier to track (and we can add other things that relate to seeing that message as well I suppose).

Reply:
Yes. See the current page. I think this was changed.